### DISCUSSION TITLE
Where we can find airflow.cfg and webserver_config.py file.

### DISCUSSION THREAD

**Discussion by Riturajvats** [apache/airflow#24686] [Q&A]
Earlier those files are present in dependency folder but now there is no folder called dependencies folder in the repo
![image_2022-06-28_015014921](https://user-images.githubusercontent.com/54302152/176028573-bf256ac6-2334-4719-abad-4e578fbc1ad2.png)
. Is that shifted to other directory ?

**Comment by alexkruc** [comment]
You can find the default configs and templates here:
https://github.com/apache/airflow/tree/main/airflow/config_templates

**Comment by potiuk** [comment]
BTW. In cas you miss it @Riturajvats  - there is a very nice "Go to file" butting that you can use if you search for a file in GitHub repo. Highly recommended.
